Sources/KIF/Additions/UIView-KIFAdditions.m

```diff
@@ -374,12 +382,25 @@ - (NSArray *)subviewsWithClassNamePrefix:(NSString *)prefix;
         if ([NSStringFromClass([view class]) hasPrefix:prefix]) {
             [result addObject:view];
         }
+        
+        if([NSStringFromClass([view class]) isEqualToString:@"_UIRemoteKeyboardPlaceholderView"]) {
```

Comment:
Rather than duplicating this logic, it'd be cleaner to add a `subviewsWithPlaceholderFallbackView` method that can be used in place of `self.subviews` in the 4 places we're iterating over `self.subviews` in this file.